patches/chrome-browser-resources-settings-clear_browsing_data_dialog-clear_browsing_data_dialog.ts.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts
+index cf55d912c89dcd72643a2233004cc17fcd0097f1..49c94aef462d711758b8afd4ec95f6daa9e4c1b2 100644
+--- a/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts
++++ b/chrome/browser/resources/settings/clear_browsing_data_dialog/clear_browsing_data_dialog.ts
+@@ -297,6 +297,7 @@ export class SettingsClearBrowsingDataDialogElement extends
+         value: () =>
+             [loadTimeData.getString('basicPageTitle'),
+              loadTimeData.getString('advancedPageTitle'),
++             loadTimeData.getString('onExitPageTitle'),
```

Comment:
We were previously calling `this.tabsNames_.push(loadTimeData.getString('onExitPageTitle'))` to do this but `this.tabNames` is private to the superclass and fails to compile with type-checking enabled.

Reply:
Removed this patch.